best practice 2/PdfBill.page

```diff
@@ -0,0 +1,166 @@
+<apex:page standardController="Invoice_Statement__c"
+    standardStylesheets="false" renderAs="pdf" sidebar="false"
+    showHeader="false" applyHtmlTag="false" applyBodyTag="false">
```

Comment:
To disable the default html tags, so if is true our html tags and style will not be applied